src/maker/api.rs

```diff
@@ -91,7 +91,7 @@ impl Maker {
         wallet_file: &PathBuf,
         rpc_config: &RPCConfig,
         port: u16,
-        onion_addrs: String,
+
```

Comment:
Seems like an extra blank? wonder why fmt isn't catching this.

Reply:
Removed